min_heap_binary_tree.rb

```diff
@@ -15,6 +15,26 @@ def insert(node)
     set_min_heap_state
   end
 
+  def delete(node)
+    # byebug
+    replacement_node = tree[-1]
+    delete_index = tree.find_index(node)
+
+    delete_node(delete_index)
+
```

Comment:
You can remove this space, to bring together that grouping of "actions taken".  So you would have a grouping of variables being set, then a space, then a group of actions being taken.